extensions/amp-story/1.0/amp-story-cta-layer.js

```diff
@@ -60,18 +60,18 @@ export class AmpStoryCtaLayer extends AmpStoryBaseLayer {
     const ctaLinks = this.element.querySelectorAll('a');
     for (let i = 0; i < ctaLinks.length; i++) {
       addAttributesToElement(ctaLinks[i],
-          /** @type {!JsonObject} */ (parseJson({'target': '_blank'})));
+          /** @type {!JsonObject} */ (parseJson(`{"target": "_blank"}`)));
```

Comment:
I think you can use `dict({...})` from `src/utils/object.js` instead of casting to `JsonObject`